### PR TITLE
add git hash and build date to MainActivity

### PR DESCRIPTION
**Pull request by oldherl**

### CODE REVIEW

**Comment by oldherl** [comment]
May be ugly implementation. Welcome any edits.

File: app/src/main/res/layout/activity_main.xml [hunk 1]
```diff
@@ -6,6 +6,16 @@
     android:layout_height="match_parent"
     tools:context=".MainActivity">
 
+    <TextView
+        android:id="@+id/version_info"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text=""
+        app:layout_constraintBottom_toTopOf="@id/open_ime_settings_guide"
```
Comment (rocka) [inline app/src/main/res/layout/activity_main.xml:14]:
```suggestion
        app:layout_constraintBottom_toBottomOf="parent"
```

File: app/src/main/res/layout/activity_main.xml [hunk 2]
```diff
@@ -6,6 +6,16 @@
...
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"/>
```
Comment (rocka) [inline app/src/main/res/layout/activity_main.xml:17]:
```suggestion
        app:layout_constraintTop_toBottomOf="@id/open_ime_settings"/>
```

File: app/src/main/res/layout/activity_main.xml [hunk 3]
```diff
@@ -14,7 +24,7 @@
         app:layout_constraintBottom_toTopOf="@id/open_ime_settings"
         app:layout_constraintLeft_toLeftOf="parent"
         app:layout_constraintRight_toRightOf="parent"
-        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintTop_toBottomOf="@id/version_info"
```
Comment (rocka) [inline app/src/main/res/layout/activity_main.xml:27]:
```suggestion
        app:layout_constraintTop_toTopOf="parent"
```

**Review by rocka — CHANGES_REQUESTED** [review]
Put version info under button

**Review by rocka — APPROVED** [review]

**Pull request merged**